scripts/ci/verify-plan.ps1

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env powershell
+
+#Requires -Version 5
+
+param(
+    # The name of the plan that is to be built.
+    [string]$Plan
+)
+
+$env:HAB_ORIGIN = 'ci'
+$Plan = 'chef-infra-client'
+
+Write-Host "--- :8ball: :windows: Verifying $Plan"
+
+Write-Host "--- :habicat: Installing the version of Habitat required"
+Install-Habitat --version 0.85.0.20190916
```

Comment:
some day we'll probably not want this lost deep in here somehow.

Reply:
Yup. I had a variant of this apparently lost in shuffling that would only attempt to install this version if the installed habitat version was below 0.85. I should bring that back. This and [the version test in `Invoke-Begin`](https://github.com/chef/chef/pull/8900/files#diff-54f992763c1ee2453d8e339101744a01R21-R27) are some early workarounds to demonstrate a possible missing piece of Habitat plan metadata: a minimum version of Habitat required for a plan to be built by.

Reply:
Updated with an "install only if hab is not installed or is too old" wrapper.

Reply:
Updated with an "install only if hab is not installed or is too old" wrapper.